README.Rmd

```diff
@@ -98,9 +98,9 @@ library(ggplot2)
 
 ### Data
 
-Nowcasting is effectively the estimation of reporting patterns for recently reported data. This requires data on these patterns for previous observations and typically this means the time series of data as reported on multiple consecutive days (in theory non-consecutive days could be used but this is not yet supported in `epinowcast`). For this quick start these data are sourced from the [Robert Koch Institute via the Germany Nowcasting hub](https://github.com/KITmetricslab/hospitalization-nowcast-hub/wiki/Truth-data#role-an-definition-of-the-seven-day-hospitalization-incidence) where they are deconvolved from weekly data and days with negative reported hospitalisations are adjusted. 
+Nowcasting of right-truncated case counts involves the estimation of reporting delays for recently reported data. This requires data on these patterns for previous observations and typically this means the time series of data as reported on multiple consecutive days (in theory non-consecutive days could be used, but this is not yet supported in `epinowcast`). For this quick start, we use data sourced from the [Robert Koch Institute via the Germany Nowcasting hub](https://github.com/KITmetricslab/hospitalization-nowcast-hub/wiki/Truth-data#role-an-definition-of-the-seven-day-hospitalization-incidence). The data represent hospitalisation counts by date of positive test and date of test report in Germany up to October 1, 2021. For this, the originally weekly data were deconvolved to a daily resolution, and days with negative counts are adjusted.
```

Comment:
Maybe we can still improve on the definition of nowcasting here and simplify a little bit.

Maybe as the second sentence:
_For this, we need to count cases both by when they occurred (often called "reference date") and by when they were reported (often called "report date"). Then, the difference between the report date and the reference date is the reporting delay._

And do we really need the remark about the non-consecutive days here? I am worried this is only confusing people